### DISCUSSION TITLE
特斯拉自動駕駛導致三人死亡車禍，NHTSA 成立專案小組調查

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#632] [General]
**美國加州上週發生一起死亡車禍，疑似因為特斯拉自動駕駛出問題，導致三人死亡。美國高速公路交通安全局今日表示，已經成立特殊調查小組，針對這起意外深入追查原因。**

看法 : 自動駕駛是最近非常流行的趨勢，未來很有可能開車不需要使用雙手，然而自動駕駛在辨別路況的時候，目前還有一些特殊的車輛或是路況無法準確辨別，目前的特斯拉還是僅輔助開車，如果太過相信自動駕駛可能會發生意外。

全文 : https://technews.tw/2022/05/20/tesla-fatal-crahs-under-investigation/

**Comment by 11timyyy** [comment]
感覺目前還是不能完全任由AI操控，自己也要時時保持警覺才行。

**Comment by henrywen547** [comment]
現在本來就沒有所謂的完全自駕車，特斯拉在整個自駕車的發展之中，甚至也不是龍頭，只是就有很多人拿特斯拉來當那種連方向盤都沒有的完全自主的自駕車在開就是了。

**Comment by Lai27330** [comment]
有些警覺度較低的人會過度依賴自動駕駛，或是可能會衍生無照駕駛的問題，因此行車駕駛還是需要時刻警覺路況才行。

**Comment by awkward-willy** [comment]
現階段的輔助駕駛很容易讓駕駛人疏忽，認為自動駕駛方便且安全
或許還得透過各式各樣的講習來加強購買人的認知，以達到更安全的交通路況

**Comment by jason79461385** [comment]
通常都是在高速公路這種地方才會用無人駕駛，如果是一般市區的話突發狀況較多比較不適合。

**Comment by liangyinglly** [comment]
或許需要更多數據使自駕車能更順利應對路況的不確定性

**Comment by HsuChen88** [comment]
特斯拉偶爾會發生一些機械上的故障，電腦系統判斷到之後只會警告，不幸的話電腦自動關閉，車上所有的東西也會關閉了。

**Comment by HsuChen88** [comment]
之前一件意外為，硬體故障發生火燒，最終電腦自行關閉，駕駛想逃生卻所有門窗都打不開的情形，或許車子不能完全建立在電腦系統上。

**Comment by HaoFish123** [comment]
感謝分享，但我聽說目前其實還未有真正的自駕車，所以上路還是要注意

**Comment by treeann** [comment]
看來離自駕車還有一段距離壓

**Comment by yucheng1028** [comment]
雖然自駕其實比大多人開車都還要安全了，但還是有一些風險

**Comment by alexthebigpp** [comment]
雖然說特斯拉已經可以很好的做出自駕了，但路上變化多端，風險還是存在。

**Comment by yenwei12138** [comment]
目前的人工智慧還無法完全取代人類的判斷，上路時還是要以人為主以電腦為輔

**Comment by Lynn12378** [comment]
沒想到太過依賴AI帶來的影響這麼大

**Comment by Jimmy911218** [comment]
感覺很方便但是如果發生意外真的會發生危險的事還是小心注意

**Comment by Hchejxnc** [comment]
感覺讓每一部車子都有與聯網技術的結合，完全自駕才有機會。

**Comment by 110409538** [comment]
覺得目前的AI技術還不夠成熟,還是有安全隱患.
